docs/changelog.md

```diff
@@ -4,6 +4,77 @@
+
+* http4s-blaze-client
+    * Fix counting of current allocated connections in the blaze client pool manager by @danicheg in https://github.com/http4s/http4s/pull/6254
+    * Roll back the #6254 by @danicheg in https://github.com/http4s/http4s/pull/6332
+
+* Documentation
+    * Release v0.21.33 by @rossabaker in https://github.com/http4s/http4s/pull/6147
+    * Update sbt-http4s-org to 0.13.1 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6197
+
+* Behind the scenes
+    * Add unidocs project to root aggregate by @armanbilge in https://github.com/http4s/http4s/pull/6129
+    * Update tomcat-catalina, tomcat-coyote, ... to 9.0.60 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6132
+    * Update http4s-circe, http4s-ember-client to 0.23.11 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6149
+    * Merge after 0.21.33 by @rossabaker in https://github.com/http4s/http4s/pull/6153
+    * Don't override api url by @armanbilge in https://github.com/http4s/http4s/pull/6158
+    * Fix StatusSpec sanitization property by @rossabaker in https://github.com/http4s/http4s/pull/6184
+    * Update sbt-doctest to 0.10.0 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6190
+    * Pin http4s-crypto to 0.1.x in 0.22 by @armanbilge in https://github.com/http4s/http4s/pull/6218
+    * Update tomcat-catalina, tomcat-coyote, ... to 9.0.62 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6214
+    * Update jetty-client, jetty-http, ... to 9.4.46.v20220331 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6220
+    * Update cats-parse to 0.3.7 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6224
+    * Update scala-xml to 2.1.0 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6234
+    * Update scalafmt-core to 3.5.1 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6256
+    * Update sbt-scalafix, scalafix-core, ... to 0.10.0 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6260
+    * Update Java-WebSocket to 1.5.3 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6270
+    * Fix deprecated since versions by @danicheg in https://github.com/http4s/http4s/pull/6279
+    * Remove ignoring some files for `doctest` by @danicheg in https://github.com/http4s/http4s/pull/6284
+    * Update sbt-http4s-org to 0.13.2 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6287
+    * Update netty-buffer, netty-codec-http to 4.1.76.Final in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6292
+    * Update scalafmt-core to 3.5.2 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6318
+    * Update fs2-core, fs2-io, ... to 2.5.11 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/6322
```

Comment:
I think Ross likes to mention some of the important update in the main notes e.g. see:
https://github.com/http4s/http4s/releases/tag/v0.23.11

So probably for core should mention fs2 and cats-parse.

Also should mentioned tomcat, jetty, scala-xml updates for their modules.

Reply:
I've polished this a bit up.